t/ui/13-admin.t

```diff
@@ -448,6 +448,29 @@ subtest 'job property editor' => sub() {
         is($driver->find_element_by_id('editor-carry-over-bugrefs')->is_selected(), 0, 'bug carry over disabled');
         is($driver->find_element_by_id('editor-description')->get_value(), 'Test group', 'description added');
     };
+
+    subtest 'edit the yaml' => sub() {
+        $driver->refresh();
+        my $form = $driver->find_element_by_id('editor-form');
+        ok($form->is_hidden(), 'editor form is hidden');
+        $driver->find_element_by_id('toggle-yaml-editor')->click();
+        ok($form->is_displayed(), 'editor form is shown');
+        ok($driver->find_element_by_id('update-template')->is_displayed(), 'update button is shown');
```

Comment:
You don't need that check because the button is clicked anyways (and selenium will fail to click on hidden elements).